### ISSUE TITLE
Doesn't work, Error with sendTextMessage

### ISSUE THREAD

**Issue by mkody** [SFTtech/sftmumblebot#17]
With a friend, we're trying to set-up the bot but it just can't join a channel on IRC and sending message to him makes some errors.

When starting with `./run`, I'm getting those lines first: (redacted username and server)
```
sft mumble bot 0.0
loading conf file sftbot.conf
(2) console:   connection successfully opened
(2) console:   initial packages successfully sent
(3) mumble:    trying python default ssl socket
(2) mumble:    connection successfully opened
(2) mumble:    initial packages successfully sent
(2) mumble:    unhandeled package received: Version, 71 bytes
(2) irc:       connection successfully opened
(3) irc:       tx: NICK <botUsername>
(3) irc:       tx: USER <botUsername> <ircServer> bla :<botUsername>
(3) irc:       tx: JOIN #<ircChannel>
(2) irc:       initial packages successfully sent
(3) irc:       rx: :c<ircServer> NOTICE Auth :*** Looking up your hostname...
(2) mumble:    unhandeled package received: CryptSetup, 54 bytes
(3) mumble:    unknown text message sender id: 0
(2) mumble:    text message received, sender: unknown
mumble: unknown: <strong>WARNING:</strong> Your client doesn't support the Opus codec the server is using, you won't be able to talk or hear anyone. Please upgrade to a client with Opus support.
(1) irc:       could not send text message: <type 'exceptions.Exception'>
(2) irc:       Traceback (most recent call last):
(2) irc:         File "sftbot/AbstractConnection.py", line 286, in sendTextMessage
(2) irc:           raise Exception("connection not established")
(2) irc:       Exception: connection not established
(2) irc:
```

When sending a message from the console:
```
(1) irc:       could not send text message: <type 'exceptions.Exception'>
(2) irc:       Traceback (most recent call last):
(2) irc:         File "sftbot/AbstractConnection.py", line 286, in sendTextMessage
(2) irc:           raise Exception("connection not established")
(2) irc:       Exception: connection not established
(2) irc:
(1) mumble:    could not send text message: <type 'exceptions.Exception'>
(2) mumble:    Traceback (most recent call last):
(2) mumble:      File "sftbot/AbstractConnection.py", line 286, in sendTextMessage
(2) mumble:        raise Exception("connection not established")
(2) mumble:    Exception: connection not established
(2) mumble:
```

If I'm sending a message from Mumble, the error only shows about IRC, and the same for IRC to Mumble.

Tested on a `Debian 7` server and on my Mac with `OS X 10.10`.

**Comment by TheJJ** [comment]
The problem seems to be that the Mumble thread does not establish a connection.
The cause seems to be:
```
(2) mumble:    unhandeled package received: CryptSetup, 54 bytes
(3) mumble:    unknown text message sender id: 0
(2) mumble:    text message received, sender: unknown
mumble: unknown: <strong>WARNING:</strong> Your client doesn't support the Opus codec the server is using, you won't be able to talk or hear anyone. Please upgrade to a client with Opus support.
```

Are you running the latest version (probably yes)?
The opus annouce is done here: https://github.com/SFTtech/sftmumblebot/blob/6025af96ee469eb6fa18a3c4c16a9ac7a6f7c6f9/sftbot/MumbleConnection.py#L98

Does the bot show up in your murmur log with the correct 1.2.6 version?

**Comment by mkody** [comment]
Our mumble server version is 1.2.8 and Opus is available. ([Screenshot](http://s.kdy.ch/wcp37.png))
The bot show up connected with the 1.2.6 version, but it says that Opus is not supported. ([Screenshot](http://s.kdy.ch/cd936.png))

**Comment by TheJJ** [comment]
Can you change the line i noted above to 0x010208 and try again?

**Comment by mkody** [comment]
Still no changes.
```
sft mumble bot 0.0
loading conf file sftbot.conf
(2) console:   connection successfully opened
(2) console:   initial packages successfully sent
(3) mumble:    trying python default ssl socket
(2) mumble:    connection successfully opened
(2) mumble:    initial packages successfully sent
(2) mumble:    unhandeled package received: Version, 71 bytes
(2) irc:       connection successfully opened
(3) irc:       tx: NICK IV_Derpy
(3) irc:       tx: USER IV_Derpy irc.XXX.org bla :IV_Derpy
(3) irc:       tx: JOIN #XXX
(2) irc:       initial packages successfully sent
(3) irc:       rx: :XXX.org NOTICE Auth :*** Looking up your hostname...
(2) mumble:    unhandeled package received: CryptSetup, 54 bytes
(3) mumble:    unknown text message sender id: 0
(2) mumble:    text message received, sender: unknown
mumble: unknown: <strong>WARNING:</strong> Your client doesn't support the Opus codec the server is using, you won't be able to talk or hear anyone. Please upgrade to a client with Opus support.
(1) irc:       could not send text message: <type 'exceptions.Exception'>
(2) irc:       Traceback (most recent call last):
(2) irc:         File "sftbot/AbstractConnection.py", line 286, in sendTextMessage
(2) irc:           raise Exception("connection not established")
(2) irc:       Exception: connection not established
(2) irc:
(2) mumble:    unhandeled package received: CodecVersion, 26 bytes
(2) mumble:    channel state package received
...
```

And it still doesn't join our IRC channel.

**Comment by TheJJ** [comment]
But it now seems to have joined your mumble server, right?
Can you join our IRC freenode#sfttech for help? We probably have more success debugging the problem there.

**Comment by TheJJ** [comment]
probably fixed by 7076ff84f1e77bbd55340fb3640cad89d7caa99a, notify me if you still have issues.
